development/build/utils.js

```diff
@@ -70,10 +70,38 @@ function getBrowserVersionMap(platforms, version) {
     const versionParts = [major, minor, patch];
```

Comment:
I forget. Was there a reason these build id changes need to be part of this PR?

Reply:
Yes. There are (were) runtime checks that ensure version is a specific format, but the format changes here. In order to get all checks to pass the version string needed to be updated in both. [The version update PR](https://github.com/MetaMask/metamask-extension/pull/25456) is a pre-req of this PR.

Reply:
:heavy_check_mark: